VERSION

```diff
@@ -1 +1 @@
-1.5.0-SNAPSHOT
+1.5.0
```

Comment:
since RPM does not allow you to have `-` in the version, we should replace the version from one that looks like `1.5.0-SNAPSHOT` to `1.5.0_SNAPSHOT` just for RPM. We will keep using `1.5.0-SNAPSHOT` in other packages such as DEB.